test/e2e/gardener/shoot/create_authenticate_delete.go

```diff
@@ -0,0 +1,445 @@
+	"net/url"
+	"os"
+	"strings"
+	"time"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	authenticationv1 "k8s.io/api/authentication/v1"
+	corev1 "k8s.io/api/core/v1"
+	apierrors "k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	utilruntime "k8s.io/apimachinery/pkg/util/runtime"
+	"k8s.io/client-go/rest"
+	"k8s.io/utils/ptr"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+
+	gardencorev1beta1 "github.com/gardener/gardener/pkg/apis/core/v1beta1"
+	"github.com/gardener/gardener/pkg/client/kubernetes"
+	"github.com/gardener/gardener/pkg/component/kubernetes/apiserverexposure"
+	"github.com/gardener/gardener/pkg/utils/retry"
+	e2e "github.com/gardener/gardener/test/e2e/gardener"
+	"github.com/gardener/gardener/test/framework"
+	"github.com/gardener/gardener/test/utils/access"
+)
+
+var _ = Describe("Shoot Tests", Label("Shoot", "default"), func() {
+	f := framework.NewGardenerFramework(e2e.DefaultGardenConfig("garden-local"))
+
+	It("Create, Authenticate, Delete", Label("authentication"), func() {
+		shoot1 := e2e.DefaultShoot("e2e-auth-one")
+		shoot1.Namespace = f.ProjectNamespace
+		shoot2 := e2e.DefaultShoot("e2e-auth-two")
+		shoot2.Namespace = f.ProjectNamespace
+
+		By("Create Shoots")
+		ctx, cancel := context.WithTimeout(parentCtx, 30*time.Minute)
+		defer cancel()
+
+		Expect(f.CreateShoot(ctx, shoot1, false)).To(Succeed())
+		Expect(f.CreateShoot(ctx, shoot2, false)).To(Succeed())
+
+		Expect(f.WaitForShootToBeCreated(ctx, shoot1)).To(Succeed())
+		Expect(f.WaitForShootToBeCreated(ctx, shoot2)).To(Succeed())
+
+		By("Verify shoot access using admin kubeconfig")
+		shoot1Client, err := access.CreateShootClientFromAdminKubeconfig(ctx, f.GardenClient, shoot1)
+		Expect(err).NotTo(HaveOccurred())
+
+		shoot2Client, err := access.CreateShootClientFromAdminKubeconfig(ctx, f.GardenClient, shoot2)
+		Expect(err).NotTo(HaveOccurred())
+
+		Eventually(func(g Gomega) {
+			g.Expect(validateShootAccess(ctx, shoot1Client.Client(), shoot1, true)).To(Succeed())
+			g.Expect(validateShootAccess(ctx, shoot2Client.Client(), shoot2, true)).To(Succeed())
+		}, "30s").Should(Succeed())
+
+		By("Verify a shoot cannot be accessed with a client certificate from another shoot")
+		shoot1NoAccessRestConfig := copyRESTConfigAndInjectAuthorization(shoot1Client.RESTConfig(), shoot2Client.RESTConfig())
+		shoot1NoAccessClient, err := client.New(shoot1NoAccessRestConfig, client.Options{})
+		Expect(err).NotTo(HaveOccurred())
+
+		shoot2NoAccessRestConfig := copyRESTConfigAndInjectAuthorization(shoot2Client.RESTConfig(), shoot1Client.RESTConfig())
+		shoot2NoAccessClient, err := client.New(shoot2NoAccessRestConfig, client.Options{})
+		Expect(err).NotTo(HaveOccurred())
+
+		Consistently(func(g Gomega) {
+			g.Expect(validateShootAccess(ctx, shoot1NoAccessClient, shoot1, false)).To(Succeed())
+			g.Expect(validateShootAccess(ctx, shoot2NoAccessClient, shoot2, false)).To(Succeed())
+		}, "10s").Should(Succeed())
```

Comment:
Would it make the tests more readable if `SetDefaultEventuallyTimeout(...)` and `SetDefaultConsistentlyDuration(...)` would be used instead of specifying the durations directly on each call?

Reply:
Wouldn't change the timeout of the entire tests suite and affect other tests as well? 
See https://github.com/onsi/gomega/blob/76132163ee4176163a07cef6fedafb19a8a347a8/gomega_dsl.go#L488 and https://github.com/onsi/gomega/blob/76132163ee4176163a07cef6fedafb19a8a347a8/gomega_dsl.go#L46-L47

Reply:
You are right with regards to the global change.